### DISCUSSION TITLE
67

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#764] [Comments]
http://jojoldu.tistory.com/67

**Comment by jojoldu** [comment]
작성자: rlawlstjd007

좋은 글 감사합니다 ^^

**Comment by jojoldu** [comment]
작성자: 지나가는개발자

오늘부터 창천향로님의 저의 멘토이십니다. ^*^ 메일 찾아오겠습니다

**Comment by jojoldu** [comment]
작성자: SISO

안녕하세요! 쏠쏠한 소셜 퍼포먼스마켓 시소(SISO) 입니다.

3초만에 스타벅스 아메리카노를 얻을수 있는 기회입니다~

현재 SISO에 소셜 로그인만 해도 추첨을 통해 스타벅스 커피를 증정해드리고 있습니다!!

SISO에 회원가입해스타벅스 커피도 받고 여러가지 프로젝트도 업로드해 수익을 챙겨가세요!!
http://si-so.co.kr/event/InfluencerEvent.html?utm_source=tistory&utm_campaign=siso_service&utm_medium=comment&utm_content=sisolauchingEvent 
 
여기서 자세한 정보 확인하실수 있습니다!!

문의 최 석민 매니저TEL : 070-4489-4755

**Comment by gonghojin** [comment]
항상 잘 읽고 갑니다!! ~

**Comment by jojoldu** [comment]
@gonghojin 매번 고맙습니다 :)
